source/clear-linux/get-started/virtual-machine-install/virtualbox.rst

```diff
@@ -38,7 +38,7 @@ Create a virtual machine in VirtualBox
 
    .. code-block:: bash
 
-      curl -O https://download.clearlinux.org/image/$(curl https://download.clearlinux.org/image/latest-images | grep live)
+      curl -O https://download.clearlinux.org/image/$(curl https://download.clearlinux.org/image/latest-images | grep 'live\.')
```

Comment:
Works great. Would you please add this note below? (It's a disclaimer.)

`.. note:: `grep` functionality may differ depending on your CLI. `

Reply:
@mvincerx , let me know if that renders correctly

Reply:
LGTM. Merging now. Thanks.